napari/utils/migrations.py

```diff
@@ -33,10 +37,11 @@ def _update_from_dict(*args, **kwargs):
                     )
                 warnings.warn(
                     trans._(
-                        "Argument {from_name} is deprecated, please use {to_name} instead. It will be removed in {version}.",
+                        "Argument {from_name} is deprecated, please use {to_name} instead. The argument {from_name} was deprecated in {since_version} and it will be removed in {version}.",
```

Comment:
If `since_version` is not mandatory, that may look weird. I think that's ok for a deprecation warning though. 

Side question do you want to quote the `{}` with `!r` ?

```suggestion
                        "Argument {from_name!r} is deprecated, please use {to_name!r} instead. The argument {from_name!r} was deprecated in {since_version} and it will be removed in {version}.",
```